src/components/modules/CodeRecovery/CodeRecovery.interactor.vue

```diff
@@ -1,6 +1,5 @@
 <template>
-  <otp-recovery
-    v-bind="$attrs"
+  <recovery-view
```

Comment:
`code-recovery-view`, name components as current module

Reply:
And without view prefix

Reply:
nope, we are using `import RecoveryView from './modules/RecoveryView';` this is different